docker-compose.yml

```diff
@@ -14,6 +14,16 @@ services:
       quorum_net:
         ipv4_address: '172.14.0.99'
 
+  eth-stats:
+    build: https://github.com/lucashenning/eth-netstats.git
+    ports:
+      - '3000:3000'
```

Comment:
Can we change these to double quotes i.e. `"` for all the other ports we have used them.